### ISSUE TITLE
Solution found: VueJS not working with create-single-spa CLI

### ISSUE THREAD

**Issue by tr4g** [single-spa/create-single-spa#231]
**Describe the bug or question**
I just followed the [Playground Guide](http://single-spa-playground.org/playground/) but used `vue` instead of react, and I just keep getting this error all the time, no matter what "solutions" I found on google.

**To Reproduce**
It took me 30 minutes to write all this, so please read it carefully.

### Create a vue project using single-spa CLI:
```sh
mkdir atest ; cd atest
create-single-spa
```
### Pick the following choices:
? Directory for new project **vue-content**
? Select type to generate **single-spa application / parcel**
? Which framework do you want to use? **vue**
? Organization name (can use letters, numbers, dash or underscore) **atest**

Vue CLI v4.5.9
? Please pick a preset:
**❯ Default ([Vue 2] babel, eslint)**
 
 ### Run the project
 ```sh
 cd vue-content
 npm run serve
 ```
 So far so good, you get http://localhost:8080 running just fine, and the CLI added some special single-spa stuff to the `main.js` file:
```
export const bootstrap = vueLifecycles.bootstrap;
export const mount = vueLifecycles.mount;
export const unmount = vueLifecycles.unmount;
```

Now it's time to do get actual single-spa magic, so open another terminal window at `atest` folder and:
 ```sh
 create-single-spa
 ```
### Now pick the following choices:
? Directory for new project **root-config**
? Select type to generate **single-spa root config**
? Which package manager do you want to use? **npm**
? Will this project use Typescript? **No**
? Would you like to use single-spa Layout Engine **No**
? Organization name (can use letters, numbers, dash or underscore) **atest**

### Follow playground instructions
Edit line 47 on `src/index.ejs`to add the vue project
```
      "imports": {
        "@atest/root-config": "//localhost:9000/atest-root-config.js",
        "@atest/vue-content": "//localhost:8080/js/app.js"
      }
```
Add it to the import section (not sure if is needed or not, tried both ways, not mentioned in the guide anyway), line 74 just before the import line already there:
```
    System.import('@atest/vue-content');
```
Now add it it to the `src/atest-root-config.jsp` file, replacing the example welcome code:
```
registerApplication({
  name: "@atest/vue-content",
  app: () =>
    System.import(
      "http://localhost:8080/js/app.js"
    ),
  activeWhen: ["/"],
});
```
### Run the root config project
```sh
cd root-config
npm start
```
Now point your browser to `http://localhost:9000` and you'll get a blank page. So you gotta check the console:
![image](https://user-images.githubusercontent.com/197329/102022213-10a1df80-3d64-11eb-9341-246c44968a70.png)
Then follow the instructions and you'll see the unfixable error.

---

**Expected behavior**
It should show the same as the Vue project. Instead, it shows a blank screen and errors on the console.

**Screenshots and/or console output**
Added above

**Comment by frehner** [comment]
Is it just because the version of systemjs that is pinned is behind the latest release? (and your update only changes it to look at the latest?)

If that's the case, then the update should be simple - just change the pinned number in the url to the latest version. 

The file is https://github.com/single-spa/create-single-spa/blob/master/packages/generator-single-spa/src/root-config/templates/src/index.ejs 

and should be a pretty simple PR if you want to do it. Thanks for figuring it out.

**Comment by tr4g** [comment]
@frehner I just took a second look and found out that the problem is the AMD extension

Just adding it fixes the problem:
```
  <script src="https://cdn.jsdelivr.net/npm/systemjs@6.7.0/dist/extras/amd.min.js"></script>
```

**Comment by frehner** [comment]
technically I think we can also even remove the `named-export` extra since it is now a part of core SystemJS as of [6.7.0](https://github.com/systemjs/systemjs/releases/tag/6.7.0)

**Comment by filoxo** [comment]
I tracked this down to my PR here: https://github.com/single-spa/create-single-spa/pull/214 I removed the amd extra and not the named-export extra as per the issue. I can submit a fix in a few.

**Comment by frehner** [comment]
> I tracked this down to my PR here: #214 I removed the amd extra and not the named-export extra as per the issue. I can submit a fix in a few.

ah good catch. Thanks @filoxo

**Comment by tr4g** [comment]
just submitted a PR

https://github.com/single-spa/create-single-spa/pull/232

**Comment by frehner** [comment]
should have been fixed by #233 

please let us know if you see any other issues.

**Comment by filoxo** [comment]
Thank you for looking into this @tr4g! Your diagnosis helped us get this fixed quickly. 

This fix is available in [create-single-spa@1.18.1](https://github.com/single-spa/create-single-spa/releases/tag/v1.18.1).
